File: CS/SchedulerGSync/SchedulerGSync/DAL/AppointmentObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Microsoft.EntityFrameworkCore;

namespace SchedulerGSync.DAL {
    public class AppointmentObject {
        
        public int Id { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public string Subject { get; set; }
        public int AppointmentType { get; set; }
        public string RecurrenceInfo { get; set; }
        public string ReminderInfo { get; set; }
        public string Location { get; set; }
        public string Label { get; set; }
        public string GId { get; set; }
        public string ETag { get; set; }
    }
}

File: VB/SchedulerGSync/SchedulerGSync/DAL/AppointmentObject.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports Microsoft.EntityFrameworkCore

Namespace SchedulerGSync.DAL
	Public Class AppointmentObject

		Public Property Id() As Integer
		Public Property Start() As DateTime
		Public Property [End]() As DateTime
		Public Property Subject() As String
		Public Property AppointmentType() As Integer
		Public Property RecurrenceInfo() As String
		Public Property ReminderInfo() As String
		Public Property Location() As String
		Public Property Label() As String
		Public Property GId() As String
		Public Property ETag() As String
	End Class
End Namespace
